packages/odyssey-storybook/src/components/Field/Field.mdx

```diff
@@ -1,43 +1,150 @@
+</Canvas>
+
+The label also indicates the Field's optional/required status to ensure screen readers properly announce the state.
+
+<Canvas>
+  <Story id="components-field--field-optional" />
+</Canvas>
+
+### Hint
+
+A hint is helper text for the Field. The field hint is similar to the disallowed `placeholder` attribute. It should provide any necessary help text for the Field but is optional.
+
+Appropriate examples include expected formatting or value types.
+
+<Canvas>
+  <Story id="components-field--field-hint" />
+</Canvas>
+
+### Error
+
+Inline error messaging for the Field.
+
+All errors require a visible or SR-only "Error:" label. This labeling is not handled automatically and must be included in the error message.
+
+<Canvas>
+  <Story id="components-field--field-error" />
+</Canvas>
+
+## Props
+
+<ArgsTable />
+
+## Variants
+
+While their appearance is identical, Field supports single and grouped inputs by offering an `as=" foo"` polymorphic API.
+
+### Single Input
+
+Fields for lone inputs are rendered as `div's and assign their `label`/`input` relationship using an ID.
+
+<Canvas>
+  <Story id="components-textarea--default" />
+</Canvas>
+
+### Grouped Inputs
+
+Fields for grouped inputs are rendered as `fieldset`s, using a `legend` to label all of the associated `input`s accessibly. This variant also spreads hint and error associations across affected controls.
+
+<Canvas>
+  <Story id="components-radio--default" />
+</Canvas>
+
+## States
+
+Fields may be either required or optional, depending on what's called for by their input.
+
+### Required
+
+By default, inputs and Fields are set to `required`. As the default, this state has no unique affordance.
+
+<Canvas>
+  <Story id="components-textinput--text" />
+</Canvas>
+
+### Optional
+
+Fields with `required` set to `false` will be marked as optional:
+
+<Canvas>
+  <Story id="components-textinput--optional" />
+</Canvas>
 
 ## Usage
 
-Field labels provide textual affordances and context for Odyssey's various inputs. Our UI Kit provides these labels alongside each component.
+Field is a complementary component, providing shared structure across Odyssey Inputs like [Text Input](?path=/docs/components-textinput--text#text-input).
 
-Please refer to individual components for complete documentation. These examples are provided for quick illustration.
+When assembling Forms, it's unnecessary to utilize Field, as it's built into Odyssey's Inputs already.
 
-### Label
+If you are building a new or bespoke Input component, please utilize Field to ensure consistent behavior with existing components.
+
+## Content Guidelines
+
+### Labels
+
+A well-written label describes the purpose of the Field for the user.
+
+Use concise, descriptive labels (a word or two) to help users quickly scan your form.
+
+Finally, please use sentence casing for your labels. Sentence case is easier to read and will help reduce error rates and speed up form completion time.
```

Comment:
an example will help here

Reply:
Examples added!

General note: I'm sticking to MDX for content examples until we have a solution for non-CSF stories or hiding "content" stories from the sidebar/VRT. Ideally, these will be in-component examples soon.